### DISCUSSION TITLE
Move out of range

### DISCUSSION THREAD

**Discussion by dhnsilva** [jschuh/klipper-macros#145] [Q&A]
Hi, I would like a light to solve this recurrent problem that appears after the update!
At the end of each print, the printer leaves some inconsistent reference that generates the 
 "**Move out of range: 455.025 463.000 295.590 [6726.162]**" error for the next print. 
 
 Every subsequent print is prevented from being executed by the "MOVE OUT OF RANGE" error and I have to restart Klipper in order to continue.
 
 The only one change in my printer was the update of KM!
 
 I had tried the Troubleshooting without sucess!
 
 Help me please!

**Comment by jschuh** [comment]
It's not really possible to diagnose something like that without seeing the log file (and the full config, although that will be in the log).
